openapi_python_client/cli.py

```diff
@@ -100,6 +100,7 @@ def handle_errors(errors: Sequence[GeneratorError]) -> None:
 def generate(
     url: Optional[str] = typer.Option(None, help="A URL to read the JSON from"),
     path: Optional[pathlib.Path] = typer.Option(None, help="A path to the JSON file"),
+    custom_template_path: Optional[str] = typer.Option(None, help="A path to a custom template directory"),
```

Comment:
What does `FileSystemLoader` do in the event the directory doesn't exist or isn't a directory? We should probably take in a `pathlib.Path` here so that Typer can do some [validation](https://typer.tiangolo.com/tutorial/parameter-types/path/#path-validations) for us.